src/deps.node.ts

```diff
@@ -3,3 +3,4 @@
 export { default as path } from "path";
 export { z } from "zod";
 export * from "ts-morph";
+export const tsMorphVersion = "12.2.0";
```

Comment:
Unnecessary. `ts.version` is sufficient and easy to obtain.

Reply:
Alternatively, perhaps it could be read from `package*.json`

Reply:
And it might change across `npm install`s, since npm allows semver-minor updates by default